docker/publish-by-arch.sh

```diff
@@ -0,0 +1,248 @@
+    inlong/dataproxy:latest-arm64 inlong/dataproxy:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-manager:latest \
+    inlong/tubemq-manager:latest-arm64 inlong/tubemq-manager:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-all:latest \
+    inlong/tubemq-all:latest-arm64 inlong/tubemq-all:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-build:latest \
+    inlong/tubemq-build:latest-arm64 inlong/tubemq-build:latest-x86
+  docker manifest create --insecure --amend inlong/dashboard:latest \
+    inlong/dashboard:latest-arm64 inlong/dashboard:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-cpp:latest \
+    inlong/tubemq-cpp:latest-arm64 inlong/tubemq-cpp:latest-x86
+  docker manifest create --insecure --amend inlong/audit:latest \
+    inlong/audit:latest-arm64 inlong/audit:latest-x86
+
+  docker manifest create --insecure --amend inlong/manager:${MVN_VERSION} \
+    inlong/manager:${MVN_VERSION}-arm64 inlong/manager:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/agent:${MVN_VERSION} \
+    inlong/agent:${MVN_VERSION}-arm64 inlong/agent:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/dataproxy:${MVN_VERSION} \
+    inlong/dataproxy:${MVN_VERSION}-arm64 inlong/dataproxy:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-manager:${MVN_VERSION} \
+    inlong/tubemq-manager:${MVN_VERSION}-arm64 inlong/tubemq-manager:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-all:${MVN_VERSION} \
+    inlong/tubemq-all:${MVN_VERSION}-arm64 inlong/tubemq-all:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-build:${MVN_VERSION} \
+    inlong/tubemq-build:${MVN_VERSION}-arm64 inlong/tubemq-build:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/dashboard:${MVN_VERSION} \
+    inlong/dashboard:${MVN_VERSION}-arm64 inlong/dashboard:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-cpp:${MVN_VERSION} \
+    inlong/tubemq-cpp:${MVN_VERSION}-arm64 inlong/tubemq-cpp:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/audit:${MVN_VERSION} \
+    inlong/audit:${MVN_VERSION}-arm64 inlong/audit:${MVN_VERSION}-x86
+  echo "End pushing manifest"
+}
+
+help() {
+  cat <<EOF
+Usage: ./publish-by-arch.sh [option] <arch>
+where arch is one of:
+  x86
+  aarch64
+Options:
+  -b        Add build operation before publish. Build docker images by arch.
+  -t        Add tag operation before publish. Add arch after version.
+  -p        Publish images according to docker registry information.
+  -m        Push manifest. This option doesn't need arch.
+  -h        Show help information.
+Example:
+  Use "./publish-by-arch.sh -b aarch64" to publish arm images after build operation.
+  Use "./publish-by-arch.sh -t x86" to publish amd images after tag already x86 images as x86.
+EOF
+}
+
+for arg in $@
```

Comment:
Double quote array expansions to avoid re-splitting elements.
```suggestion
for arg in "$@"
```